gatsby/src/components/schema/schema.tsx

```diff
@@ -96,6 +106,21 @@ export const SchemaComp: React.FC<IProps> = ({
     dateModified = datePublished || null;
   }
 
+  let faqList = null;
+  if (typeof questions_answers !== 'undefined') {
+    faqList = [];
+    questions_answers.forEach((questions_answer) => {
+      faqList.push({
+        '@type': 'Question',
+        name: questions_answer.question,
+        acceptedAnswer: {
+          '@type': 'Answer',
+          text: questions_answer.value,
+        },
+      });
+    });
+  }
```

Comment:
```suggestion
  const faqList = questions_answers?.map((questions_answer) => ({
  '@type': 'Question',
  name: questions_answer.question,
  acceptedAnswer: {
    '@type': 'Answer',
    text: questions_answer.value,
  },
}));
```
Would probably just work as well? :) and its much cleaner.

Reply:
Yeah, it works too and it is a lot cleaner, you are right.